CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 #### 0.12.3 (Next)
 
 * Your contribution here.
+* * [#127](https://github.com/slack-ruby/slack-ruby-bot-server/pull/127): Added a Rails bot starter built on top of the slack ruby bot server - [@CrazyOptimist](https://github.com/CrazyOptimist).
```

Comment:
One * too many hence the bot complains. Either way no need for a changelog line for a README update.

Reply:
![image](https://user-images.githubusercontent.com/55074937/94469407-b002fd80-018b-11eb-9383-c33a7b539387.png)
Build failure is not due to the changelog line I guess.
Shall I remove the changelog line anyhow?

Reply:
Oops sorry,
Just fixed my typo.